### PR TITLE
Updates relating to obsolete pkg_add(1), pkg_create(1) and pkg_info(1)

### PR DESCRIPTION
**Pull request by grahamperrin**
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=274294

### CODE REVIEW

**Review by igalic — CHANGES_REQUESTED** [review]
I think all the calls to pkg-add(8) should probably be calls to pkg-install(8)

**Comment by grahamperrin** [comment]
> I think all the calls to pkg-add(8) should probably be calls to pkg-install(8)

Nice catch! Thanks. 

For myself, I need to take a closer look, so I'll leave this PR in the draft state for maybe a few days. 

References: 

* [pkg_add(1)](https://man.freebsd.org/cgi/man.cgi?query=pkg_add&sektion=1&manpath=FreeBSD+9.3-RELEASE) in FreeBSD 9.3-RELEASE
* [pkg_add is no more the default tool, make motd recommand using pkg install instead of pkg_add · freebsd/freebsd-src@2394542](https://github.com/freebsd/freebsd-src/commit/2394542b4e2455141818936c11734c82275aedab) on branch `release/10.0.0`
* [pkg-add(8)](https://man.freebsd.org/cgi/man.cgi?query=pkg-add&sektion=8&manpath=freebsd-ports) in ports
* [pkg-install(8)](https://man.freebsd.org/cgi/man.cgi?query=pkg-install&sektion=8&manpath=freebsd-ports) in ports
* [pkg-register(8)](https://man.freebsd.org/cgi/man.cgi?query=pkg-register&sektion=8&manpath=freebsd-ports) in ports.

File: usr.sbin/pkg/pkg.7 [hunk 1]
```diff
@@ -324,7 +324,4 @@ command first appeared in
 It became the default package tool in
 .Fx 10.0 ,
 replacing the
-pkg_install suite of tools
-.Xr pkg_add 1 ,
-.Xr pkg_info 1 and
-.Xr pkg_create 1 .
+pkg_install suite of tools pkg_add(1), pkg_info(1) and pkg_create(1).
```
Comment (grahamperrin) [inline usr.sbin/pkg/pkg.7:327]: Given this line's HISTORY context in FreeBSD-RELEASE, <https://man.freebsd.org/cgi/man.cgi?query=pkg&sektion=7&manpath=freebsd-release#HISTORY>, I reckon that this change is OK.

File: usr.sbin/bsdinstall/scripts/docsinstall [hunk 2]
```diff
@@ -148,7 +148,7 @@ f_mustberoot_init
 dialog_menu_main || f_die
 f_dialog_menutag_fetch selected
 
-# Let pkg_add be able to use name servers
+# Let pkg-add be able to use name servers
```
Comment (grahamperrin) [inline usr.sbin/bsdinstall/scripts/docsinstall:151]: In the `bsdinstall` context, what's the proper way to rethink this comment? 

Whilst I _can_ imagine something using name servers, I can _not_ imagine it being anything, from a package server, that precedes a bootstrap by [pkg(7)](https://man.freebsd.org/cgi/man.cgi?query=pkg&sektion=7&manpath=freebsd-release) in base.

(Does that make sense?)

Thanks @igalic for encouraging me to think twice about this PR.

File: usr.sbin/pkg/pkg.7 (hunk 1 above)
Comment (emaste) [reply usr.sbin/pkg/pkg.7:327]: There is no `pkg_add` page in section 1 though, so I think there's no value in adding the section number suffix.

**Comment by bsdimp** [comment]
This has been a draft for months. Please finish or close soon.

**Comment by bsdimp** [comment]
I'm going to go ahead and close this to reduce the clutter. However, once you've finished the revision, please reopen or create a new pull request. Thanks!

**Pull request closed without merging**